src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,68 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { SortType } from '../../types/Filter';
+
+type Props = {
+  sortType: SortType;
+  completeItem: number;
+  onSortChange: (sortType: SortType) => void;
+};
+
+export const Footer: FC<Props> = ({
```

Comment:
Better rename to `Filter`.

Reread [Task checklist point 3](https://github.com/mate-academy/react_todo-app-loading-todos/blob/master/checklist.md). Component names examples there more meaningful.

Be always attentive with tasks descriptions and checklists. There **no redundant** words. The same would be in test tasks. You need to read it attentively and write the code according to it. It is important to pass all employment stages and get the job 😉